### PR TITLE
PLAT-145625: Fix VideoPlayer to handle playback rate with decimal point

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fix `VideoPlayer` to handle playback rate with decimal point 

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Use parseFloat instead of parseInt in `calcNumberValueOfPlaybackRate` func

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-145625

### CODE REVIEW

File: samples/sampler/stories/qa/VideoPlayer.js [hunk 1]
```diff
@@ -97,3 +99,50 @@ export default {
...
+				>
+					<Video>
+						<source src={'http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4'} />
+					</Video>
+					<MediaControls actionGuideLabel="Press down button to fastforward">
+						<Button
+							icon="forward"
+							onClick={this.fastforward}
+						/>
+					</MediaControls>
```
Comment (jeonghee27) [inline samples/sampler/stories/qa/VideoPlayer.js:139]: Could you please add other buttons for "rewind" , "slowForward", and "slowRewind"?

File: VideoPlayer/VideoPlayer.js [hunk 2]
```diff
@@ -156,15 +156,15 @@ const AnnounceState = {
  * faster. If it is negative, it will play backward.
  *
  * The order of numbers represents the incremental order of rates that will be used for each
- * operation. Note that all rates are expressed as strings and fractions are used rather than decimals
- * (e.g.: `'1/2'`, not `'0.5'`).
+ * operation. Note that rates can be expressed as decimals, strings and also fractions if needed.
```
Comment (jeonghee27) [inline VideoPlayer/VideoPlayer.js:159]:
```suggestion
 * operation. Note that rates can be expressed as decimals, strings, and also fractions if needed.
```

File: VideoPlayer/VideoPlayer.js [hunk 3]
```diff
@@ -527,10 +527,10 @@ const VideoPlayerBase = class extends Component {
...
 		playbackRateHash: PropTypes.shape({
-			fastForward: PropTypes.arrayOf(PropTypes.string),
-			rewind: PropTypes.arrayOf(PropTypes.string),
-			slowForward: PropTypes.arrayOf(PropTypes.string),
-			slowRewind: PropTypes.arrayOf(PropTypes.string)
+			fastForward: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
+			rewind: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
+			slowForward: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
+			slowRewind: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number]))
 		}),
```
Comment (jeonghee27) [inline VideoPlayer/VideoPlayer.js:535]: Could you please fix L1398

	 * Changes the playback speed via [selectPlaybackRate()]{@link sandstone/VideoPlayer.VideoPlayer#selectPlaybackRate}.
	
==>

	 * Changes the playback speed.

File: VideoPlayer/VideoPlayer.js (hunk 2 above)
Comment (juwonjeong) [reply VideoPlayer/VideoPlayer.js]: It's fixed.

File: VideoPlayer/VideoPlayer.js (hunk 3 above)
Comment (juwonjeong) [reply VideoPlayer/VideoPlayer.js:535]: It's fixed. Also L1377, L1455 are fixed.

File: samples/sampler/stories/qa/VideoPlayer.js (hunk 1 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/VideoPlayer.js]: I added rewind button. "slowForward" and "slowRewind" can be tested when video is paused and fastforward or rewind button is clicked.

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Review by 0x64 — APPROVED** [review]

**Pull request merged**